core/templates/components/profile-link-directives/profile-link-image.component.html

```diff
@@ -1,5 +1,9 @@
 <a *ngIf="isUsernameLinkable(username)" [href]="profileUrl" [smartRouterLink]="profileUrl">
-  <img [src]="profilePicture" alt="User Avatar" class="rounded-circle">
+  <picture>
+    <source type="image/webp" [srcset]="profilePictureWebpDataUrl">
+    <source type="image/png" [srcset]="profilePicturePngDataUrl">
+    <img [src]="profilePicturePngDataUrl" class="rounded-circle" alt="User Avatar">
+  </picture>
 </a>
 
-<img *ngIf="!isUsernameLinkable(username)" [src]="profilePicture" alt="">
+<img *ngIf="!isUsernameLinkable(username)" [src]="profilePictureWebpDataUrl" alt="">
```

Comment:
I don't understand this, it should either be a `<picture>` or not here.

Reply:
I'm a bit confused too, I mean do we even require an extra image here?

Reply:
Not sure, you will have to investigate the code.